DESCRIPTION

```diff
@@ -31,6 +30,7 @@ Suggests:
     attachment,
     rhub,
     rsconnect,
+    esqlabs.handsontable,
```

Comment:
Move to Imports section as it is impossible to use the app without this. Suggest is for development packages or for optional dependencies only.

Reply:
+ Add an entry `esqLABS/esqlabs.handsontable` to the remote field below.

By the way, this:
```
    datamods@dev-2,
    yonicd/covrpage
```
can be removed now